### DISCUSSION TITLE
Connect to address book

### DISCUSSION THREAD

**Discussion by bodryj** [rustdesk/rustdesk#570] [Q&A] [answered]
I have docker version running on ubuntu 20.04. When switching to the address book, error 10061. How should the address book work in general? What username and password do you mean?
![image](https://user-images.githubusercontent.com/61440472/169549433-21ac74e1-f51a-4dae-a056-b2bc9d706c95.png)

**Comment by rustdesk** [comment, marked as answer]
Server side is not ready yet for the time being. It may be a pro feature in the future.

**Reply by rustdesk** [reply]
both for self hosting and our public, please follow our twitter for latest update.

**Reply by tarakan1983** [reply]
Hello, tell me, are there any changes with the address book?

**Reply by bmcgonag** [reply]
Also curious if there's any update on this functionality.  Thanks for the amazing software btw.

**Reply by 3gmgithub** [reply]
Any update on this????

**Reply by audioscavenger** [reply]
same here, is login underway? i would gladly use it to have favorites

**Reply by screege** [reply]
Any update on how to use the address book? BTW thanks for the great software!

Regards

**Reply by rustdesk** [reply]
no update, because our all resources are put on flutter desktop rewriting.

**Reply by ranomier** [reply]
Guys the software is free and often written in free time.
What makes it worse is that this software got popular fast. As far as i know they are overwhelmed.

So cut them some slack!

**Reply by jctmf** [reply]
Thanks for this great software!

**Reply by RyanSawyer6003** [reply]
Thanks for the awesome software.

**Reply by schneider82** [reply]
I'm a corporate client looking for a better solution than TeamViewer. This looks great, but I have a lot of clients and want to use the address book. Anything on our server should be available for my teams to access. Can I help get this moving? TeamViewer want their pound of flesh, which is much more than it's truly worth, and I would prefer to give that to you guys to build the features.

**Reply by rustdesk** [reply]
soon

**Reply by schneider82** [reply]
> soon

Have you guys got a road map? Or even what these features will enable for "self-hosted" (I wish you would charge a license fee!)

**Reply by ultimatejedi** [reply]
I second this all!  Love the software!  Would also love the address book.  Currently saving all of my connections as favorites, but does not allow me to move across machines.  If I can provide assistance in the address book feature, I would be willing!

**Comment by maumaujo** [comment]
I've just installed a Rust Desk server, and I'm trying to access the Address Book, but the Linux root password doesn't work.

What is the default username and password for first login?

I already tried user admin and password test1234

![Rustdesk Address Book](https://user-images.githubusercontent.com/115227968/194546349-ab4a24ad-aa23-400a-8145-20367a835110.png)

**Reply by sgregori** [reply]
Hi mate, I think that part hasn't been published yet, it may not even be developed.

**Reply by maumaujo** [reply]
I'm testing the functionality of the Rustdesk tool, to deploy in the company.

In other posts, they reported that the Address Book function is paid, but I'm testing it before buying it.

Please request access instructions, for temporary testing.

**Comment by poisonkit** [comment]
are there php scripts for publishing on a web server? Looking for examples of work.

**Comment by Croydon** [comment]
I think the major problem here is communication. If I understand this correctly, the UI is basically a mock-up but the features are not implemented yet.
